nornir_infrahub/plugins/inventory/infrahub.py

```diff
@@ -7,7 +7,7 @@
 import ruamel.yaml
 from infrahub_sdk import Config, InfrahubClientSync
 from infrahub_sdk.node import InfrahubNodeSync
-from infrahub_sdk.schema import NodeSchema
+from infrahub_sdk.schema import NodeSchemaAPI
```

Comment:
Doing this change, means we have to update the min version of the infrahub-sdk dependency.
Also we'll need to make a release and therefor bump the package version.